packages/trader/src/sass/app/_common/components/market-symbol-icon.scss

```diff
@@ -1,5 +1,6 @@
 /** @define .market-symbol-icon; weak */
 .market-symbol-icon {
+    pointer-events: none;
```

Comment:
The market symbol has a popover / tooltip component used to display the names of the market, using `pointer-events: none` will break the `hover` / `mouseEnter` event.

Reply:
yes you were right managed to use cursor: none and solved he issue can you approve pls